package.json

```
@@ -7,16 +7,17 @@
     "build": "node scripts/generate-pwa-icons.mjs && next build",
     "dev": "next dev",
     "lint": "eslint .",
+    "start": "next start",
```

Biome (2.5.0)

[error] The key start was already declared.

(lint/suspicious/noDuplicateObjectKeys)